client/components/common/header/Header.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+import { IndexLink } from 'react-router';
+import { connect } from 'react-redux';
+import toastr from 'toastr';
+import MainMenu from './MainMenu.jsx';
+import { logout } from '../../../actions/userActions';
+
+class Header extends Component {
+  constructor(props) {
+    super(props);
+
+    this.logout = this.logout.bind(this);
+  }
+
+  static componentWillMount() {
+    $('.dropdown-button').dropdown();
+  }
+
+  logout(event) {
+    event.preventDefault();
+    this.props.logout();
+    toastr.info('You are now logged out');
+    this.context.router.push('/');
+  }
+
+  render() {
+    const loggedIn = this.props.loggedIn;
+    let isAdmin;
+    if (loggedIn) {
```

Comment:
Did you try out the ternary operator? `const isAdmin = loggedIn ? this.props.isAdmin.roleId : 0;`